### PR TITLE
chore: remove calls to pdb endpoint

### PR DESCRIPTION
**Pull request by jamiezieziula**
Screenshot 2024-09-08 at 4 51 59 PM

### CODE REVIEW

File: pkg/sloop/ingress/kubewatcher.go [hunk 1]
```diff
@@ -122,7 +122,6 @@ func (i *kubeWatcherImpl) startWellKnownInformers(kubeclient kubernetes.Interfac
 	i.informerFactory.Core().V1().PersistentVolumeClaims().Informer().AddEventHandler(i.getEventHandlerForResource("PersistentVolumeClaim", enableGranularMetrics))
 	i.informerFactory.Core().V1().PersistentVolumes().Informer().AddEventHandler(i.getEventHandlerForResource("PersistentVolume", enableGranularMetrics))
 	i.informerFactory.Core().V1().Pods().Informer().AddEventHandler(i.getEventHandlerForResource("Pod", enableGranularMetrics))
-	i.informerFactory.Policy().V1().PodDisruptionBudgets().Informer().AddEventHandler(i.getEventHandlerForResource("PodDisruptionBudget", enableGranularMetrics))
```
Comment (mitchnielsen) [inline pkg/sloop/ingress/kubewatcher.go:125]: 🤔 The error report said sloop was polling for pdbs in `v1beta1`, but here we're looking at `v1` which looks right based on the [pdb docs](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).

Comment (mitchnielsen) [reply pkg/sloop/ingress/kubewatcher.go:125]: Ah, the screenshot shows the call was made on August 30, and Jimi updated PDB checks to v1 in https://github.com/mitchnielsen/sloop/pull/2.

Comment (mitchnielsen) [reply pkg/sloop/ingress/kubewatcher.go:125]: And then in https://github.com/PrefectHQ/platform/pull/7346, he updated the reference in the Platform project to point to that new version of sloop. I believe we're just using the `latest` image tag for sloop at the moment, so it propogated in the clusters some time after that.

I bet we won't see this error anymore and can clear it from the cloud console (if possible).

Comment (jamiezieziula) [reply pkg/sloop/ingress/kubewatcher.go:125]: ah i see, thats my bad! I got this confused with something other resource that is deprecated completely. ignore me :)

**Pull request closed without merging**